### PR TITLE
Interop integration

### PR DESCRIPTION
**Pull request by cyrilderamos**
addition of taggingTab documentation

### CODE REVIEW

File: documentation/for_developers.rst [hunk 1]
```diff
@@ -57,4 +57,26 @@ Targets tab must provide an interface to update the tab when a table in the data
...
+**Tags**
+The tags created by the user are stored here for easy access. The tags can be sorted by their following characteristics: Type, Subtype, Current (Total), and Icon. Tags can be edited or removed in this section, and will also be removed on the big map showing all tags. Tags can also be added to images and new additions will be added here.
+
+**Images**
+The Images section shows all the images taken and they can come in in real time. The images are seperated into three main groups: images that have been reviewed, images that have not been reviewed, and all images. 
+
+**Map**
+If an image is selected for review, it is put onto the Map where the user can edit tags in the image
+
+What controls there are
```
Comment (yerdnayerdna) [inline documentation/for_developers.rst]: It has more controls -- take a look at the context menu

File: documentation/for_developers.rst [hunk 2]
```diff
@@ -57,4 +57,26 @@ Targets tab must provide an interface to update the tab when a table in the data
 - Tag was edited;
 - Tag was deleted;
 - New marker was created;
-- Marker was deleted;
\ No newline at end of file
+- Marker was deleted;
+
+Tagging Tab
+-----------
+The tagging tab allows the user to tag objects in images and its full description is below.
```
Comment (yerdnayerdna) [inline documentation/for_developers.rst]: Suggestion: The tagging tab allows the user to review images and tag objects of interest

File: documentation/for_developers.rst [hunk 3]
```diff
@@ -57,4 +57,26 @@ Targets tab must provide an interface to update the tab when a table in the data
...
+~~~~~~~~~~~~~
+
+**Tags**
+The tags created by the user are stored here for easy access. The tags can be sorted by their following characteristics: Type, Subtype, Current (Total), and Icon. Tags can be edited or removed in this section, and will also be removed on the big map showing all tags. Tags can also be added to images and new additions will be added here.
+
+**Images**
+The Images section shows all the images taken and they can come in in real time. The images are seperated into three main groups: images that have been reviewed, images that have not been reviewed, and all images. 
+
+**Map**
+If an image is selected for review, it is put onto the Map where the user can edit tags in the image
```
Comment (yerdnayerdna) [inline documentation/for_developers.rst]: "...put onto the Map where the user can edit tags in the image" -- what do you mean?

File: documentation/for_developers.rst (hunk 1 above)
Comment (cyrilderamos) [reply documentation/for_developers.rst]: Added two more things

File: documentation/for_developers.rst (hunk 3 above)
Comment (cyrilderamos) [reply documentation/for_developers.rst]: Clarified

File: documentation/for_developers.rst (hunk 2 above)
Comment (cyrilderamos) [reply documentation/for_developers.rst]: Implemented!

File: documentation/for_developers.rst [hunk 4]
```diff
@@ -73,10 +73,16 @@ The tags created by the user are stored here for easy access. The tags can be so
...
-If an image is selected for review, it is put onto the Map where the user can edit tags in the image
+If an image is selected for review, it is displayed onto the right portion of the window where the user can add or remove tags on the image.
 
 What controls there are
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 **Adding Tags**
 To add tag an object in the image, first click add under the Tags section and fill in corresponding characteristics. Then hover over object in the image and right click to tag it. 
+
+**Determining Geolocation**
```
Comment (yerdnayerdna) [inline documentation/for_developers.rst]: Great thing to mention. It is a property of the Main Window however -- speaking of which, we should look into making a section for it this Saturday. For now, I suggest you save this section on your laptop, remove it from this commit, and we add it where it belongs on Saturday.

Comment (cyrilderamos) [reply documentation/for_developers.rst]: It has been removed

**Pull request merged**